### PR TITLE
Dyn cs field template

### PR DESCRIPTION
**Pull request by bence-nagy**

### CODE REVIEW

File: bin/assets/cloudstorm.css [hunk 1]
```diff
@@ -38,18 +38,43 @@
...
+  */
+  /*
+  * Horizontal align
+  */ }
+  .cs-field .cs-field-inner {
+    position: relative;
+    padding-bottom: 15px; }
+  .cs-field .hidden {
+    display: none; }
+  .cs-field .cont_v {
```
Comment (bence-nagy) [inline bin/assets/cloudstorm.css]: Please use param-case for css classes and more meaningful names (e.g.: cont_v -> vertical-layout, field1_v -> label-vertical, field2_h -> value-horizontal or content-horizontal, labelContainer -> label-container, etc.)

File: src/components/cs-field/cs-field.js.coffee [hunk 2]
```diff
@@ -81,7 +80,30 @@ app.directive "csField", ['$compile', '$templateRequest', ($compile, $templateRe
...
+            field1 : "field1_v",
+            field2 : "field2_v"
+        when "create"
+          styleMap =
+            star : "show",
+            container : "container_v",
+            field1 : "field1_v",
+            field2 : "field2_v"
+        when "show"
+        	styleMap =
```
Comment (bence-nagy) [inline src/components/cs-field/cs-field.js.coffee]: Fix indentation.

File: src/components/cs-field/cs-field.js.coffee [hunk 3]
```diff
@@ -81,7 +80,30 @@ app.directive "csField", ['$compile', '$templateRequest', ($compile, $templateRe
...
+            star : "hidden",
+            container : "cont_h",
+            field1 : "field1_h",
+            field2 : "field2_h"
+
+    $scope.descriptor =
```
Comment (bence-nagy) [inline src/components/cs-field/cs-field.js.coffee]: The word 'descriptor' seems a bit overloaded in the context of Singham, maybe use 'layoutSettings' or simply 'styleMap' and drop the inner 'style:' part.

**Comment by kdaveed** [comment]
@bence-nagy Corrections are done.

**Pull request merged**